src/components/binding/binding-loader.js

```diff
@@ -247,7 +247,7 @@ genotet.BindingLoader.prototype.updateRanges_ = function() {
 };
 
 /**
- * Loads binding data list into genotet.data.bindingGenes.
+ * Loads binding data list into genotet.data.bindingFiles.
```

Comment:
It should be bindingFiles at here I think, but it is related to too many areas, I am going to fix it later.